### PR TITLE
net-irc/inspircd 3.3.0

### PR DESCRIPTION
**Pull request by clinew**
v2.0.25 has the 'x86' keyword stabilized, while v3.2.0 does not yet, hence I chose to leave it in until stablization is complete.

v2.0.28 is the LTS version, and migration to the v3.x series is non-trivial so I believe it important to include the v2.0.x series.

### CODE REVIEW

File: net-irc/inspircd/inspircd-3.3.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+	sqlite? ( >=dev-db/sqlite-3.0 )
+	ssl? ( dev-libs/openssl:= )
+	tre? ( dev-libs/tre )"
+DEPEND="${RDEPEND}"
+
+DOCS=( docs/. )
+PATCHES=( "${FILESDIR}"/${P}-fix-path-builds.patch )
+
+pkg_setup() {
+	enewgroup ${PN}
```
Comment (juippis) [inline net-irc/inspircd/inspircd-3.3.0.ebuild]: This and `enewuser` below need to be handled via the GLEP-81 way now. There are many examples in the gentoo-dev mailing list how to do it.

**Comment by juippis** [comment]
All of your commits need to be signed-off, you can do it easily by rebasing. 

I'd also urge you to switch the orders of these commits, so acct-user/inspircd comes before bumping to 3.3.0 (since 3.3.0 depends on that). So the tree is broken between these commits. It can also be done within the rebase editor.
Rest looks good.

**Comment by clinew** [comment]
Blah, sorry, I had forgotten to add the signed-off-by tag to the new commits.

The `acct-user/inspircd` commit was actually always before the `3.3.0` commit in `git`; not sure why GitHub didn't pick up on that.  Very strange.

It should be good now.

**Comment by juippis** [comment]
Thanks!

**Comment by clinew** [comment]
And thank you for the review & merge!

**Pull request closed without merging**